terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/codelist/CodeListInterceptor.java

```diff
@@ -201,17 +154,6 @@ public void setCodeListIdPattern(Pattern codeListIdPattern) {
         this.codeListIdPattern = codeListIdPattern;
     }
 
-    /**
-     * Sets the default locale to fall back.
-     * <p>
-     * this locale is used if the requested locale is not found in i18nCodeList.
-     * </p>
-     * @param fallbackTo default locale used if the requested locale is not found in i18nCodeList
-     */
-    public void setFallbackTo(Locale fallbackTo) {
-        this.fallbackTo = fallbackTo;
-    }
-
```

Comment:
- [ ] Please add JavaDoc description about role of resolving locale has bean moved to `I18nCodeList`.

Reply:
Modified by 1203474